pkg/bff/parallel.go

```diff
@@ -36,8 +37,46 @@ func (s *Server) ParallelGDSRequests(ctx context.Context, rpc RPCFunc, flatten b
 	}
 
 	// execute both requests
-	go closure(s.testnet, 0, testnet)
-	go closure(s.mainnet, 1, mainnet)
+	go closure(s.testnet.gds, 0, testnet)
+	go closure(s.mainnet.gds, 1, mainnet)
+	wg.Wait()
+
+	// flatten rpc and error if requested
+	if flatten {
+		return FlattenResults(results), FlattenErrs(errs)
+	}
+	return results, errs
+}
+
+// MembersRPC allows the BFF to issue arbitrary members client methods in parallel to both the
+// testnet and the mainnet. The members client and network are passed into the function,
+// allowing the RPC to make any GDS RPC call and log with the associated network.
+type MembersRPC func(ctx context.Context, client members.TRISAMembersClient, network string) (proto.Message, error)
+
+// ParallelMembersRequest makes concurrent requests to both the testnet and the mainnet,
+// storing the results and errors in a slice of length 2 ([testnet, mainnet]). If the
+// flatten bool is true, then nil values are removed from the slice (though this will
+// make which network returned the result ambiguous).
+func (s *Server) ParallelMembersRequests(ctx context.Context, rpc MembersRPC, flatten bool) (results []proto.Message, errs []error) {
```

Comment:
Is the only difference here the client interface? If so, we could probably just create a super client interface combining both the interfaces into one, then embedding the clients into the `NetworkClient` so that it acted as clients to both.